tensorflow_blade/.bazelrc

```diff
@@ -9,24 +10,45 @@ test --test_output=errors
 build --verbose_failures
 build --spawn_strategy=local  # this is a workaround to include third party source files with BUILD.bazel.
 
+# basic project setting
+build --cxxopt=-std=c++14
+build --host_cxxopt=-std=c++14
+build --cxxopt=-DBUILD_WITH_BAZEL
+build --action_env BUILD_WITH_BAZEL=1
+build:cxx11abi_0 --cxxopt=-D_GLIBCXX_USE_CXX11_ABI=0
+build:cxx11abi_1 --cxxopt=-D_GLIBCXX_USE_CXX11_ABI=1
+build:release --compilation_mode=opt
+build:debug --compilation_mode=dbg --strip=never --copt=-O0
+
 # these comes from tensorflow's source code
-# base
 build:release_base --distinct_host_configuration=false
-# cpu
 build:avx_linux --copt=-mavx
 build:avx_linux --host_copt=-mavx
-
 build:release_cpu_linux --config=release_base
 build:release_cpu_linux --config=avx_linux
 build:release_cpu_linux --crosstool_top="@ubuntu18.04-gcc7_manylinux2010-cuda11.2-cudnn8.1-tensorrt7.2_config_cuda//crosstool:toolchain"
-# cuda
+
+# CUDA
```

Comment:
Maybe import .bazelrc from `tf_community`? I have already make these duplicated definitions for `tao_bridge` which are copied from `tf_community`'s bazelrc

Reply:
They ARE copied from tao_bridge ... But one day `tf_community/.bazelrc` will be imported, cause `tao_compiler_main` will be built from `tensorflow_blade` workspace.